### PR TITLE
Bug fix: <v-switch> events

### PR DESCRIPTION
**Pull request by mrloureed**
This fix is related to https://github.com/viamrobotics/prime/pull/292

### CODE REVIEW

File: packages/legacy/src/elements/switch.svelte [hunk 1]
```diff
@@ -26,6 +26,7 @@ $: isReadonly = htmlToBoolean(readonly, 'readonly');
 
 const handleClick = (event: Event) => {
   if (!(isDisabled || isReadonly)) {
+    console.log(event);
```
Comment (micheal-parks) [inline packages/legacy/src/elements/switch.svelte:29]: ✂️

**Review by micheal-parks — APPROVED** [review]

File: packages/legacy/src/elements/switch.svelte (hunk 1 above)
Comment (mrloureed) [reply packages/legacy/src/elements/switch.svelte]: 🪨

Comment (micheal-parks) [reply packages/legacy/src/elements/switch.svelte]: 🧻

Comment (mrloureed) [reply packages/legacy/src/elements/switch.svelte]: 💀

**Pull request merged**